src/main/java/view/UserPageImpl.java

```diff
@@ -1,35 +1,53 @@
 package view;
 
 import model.BlindHorizontal;
-import service.HorizontalService;
-import service.HorizontalServiceImpl;
-import service.PriceCatalog;
+import model.BlindVertical;
+import service.*;
 
+import java.io.File;
 import java.util.Scanner;
 
 public class UserPageImpl implements UserPage {
 
-    private PriceCatalog priceCatalogImpl;
     private HorizontalService horizontalServiceImpl;
+    private VerticalService verticalServiceImpl;
+    private Property property = new PropertyImpl();
+    private DBBlinds db = new DBBlindsImpl();
 
-    public UserPageImpl(HorizontalService horizontalServiceImpl, PriceCatalog priceCatalogImpl) {
+    public UserPageImpl(HorizontalService horizontalServiceImpl, VerticalService verticalServiceImpl) {
         this.horizontalServiceImpl = horizontalServiceImpl;
-        this.priceCatalogImpl = priceCatalogImpl;
+        this.verticalServiceImpl = verticalServiceImpl;
     }
 
     Scanner scanner = new Scanner(System.in);
     Scanner scannerStr = new Scanner(System.in);
+    String fileName = property.getPropertyValue("outputFile");
 
     @Override
     public void showBlindCost() {
-        priceCatalogImpl.initDate();
+        File file = new File(fileName);
+        file.delete();
         while (true) {
             System.out.println("Would you like to calculate the cost of blinds?\nIf yes - enter Y, if no - enter N");
             String answer = scannerStr.nextLine();
             if (answer.equalsIgnoreCase("Y")) {
-                long costBlinds = horizontalServiceImpl.calculateCost(requestData());
-                System.out.println("Horizontal blind costs " + costBlinds + " rubles.\n");
+                long costBlinds = 0;
+                System.out.println("Choose the type of blinds: horizontal (enter H) or vertical (enter V)");
+                String horizontalOrVertical = scannerStr.nextLine();
+                if (horizontalOrVertical.equalsIgnoreCase("H")) {
+                    costBlinds = horizontalServiceImpl.calculateCost(requestDataHorizontalBlind());
+                    System.out.println("Horizontal blind costs " + costBlinds + " rubles.\n");
+                } else if (horizontalOrVertical.equalsIgnoreCase("V")) {
+                    costBlinds = verticalServiceImpl.calculateCost(requestDataVerticalBlind());
+                    System.out.println("Vertical blind costs " + costBlinds + " rubles.\n");
+                }
+                db.saveOrders(costBlinds + "\n");
             } else if (answer.equalsIgnoreCase("N")) {
+                System.out.println("To save orders to a file, entered S");
+                String answerSave = scannerStr.nextLine();
+                if (answerSave.equalsIgnoreCase("S")) {
+                    db.getOrders(fileName);
```

Comment:
This method writes data to file. Please, rename it accordingly

Reply:
done